src/getInput/getTryCount.js

```diff
@@ -0,0 +1,6 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export async function getTryCount() {
+  const TRY_COUNT = await Console.readLineAsync('시도할 횟수는 몇 회인가요?\n');
+  return TRY_COUNT;
```

Comment:
바로 return 하는 방법은 어떠신가요? 함수명으로 try count 값을 return 할 거라고 알려주고 있어서 변수로 한 번더 표현해주지 않아도 된다고 생각하거든요!

```suggestion
  return Console.readLineAsync('시도할 횟수는 몇 회인가요?\n');
```

Reply:
해당 파일에서는 굳이 변수로 받지 않아도 되었네요! 감사합니당